pkg/supportbundle/supportbundle.go

```diff
@@ -119,17 +119,13 @@ func GetSpecSecretName(appSlug string) string {
 	return fmt.Sprintf("kotsadm-%s-supportbundle", appSlug)
 }
 
-func GetSpecURI(appSlug string) string {
-	return fmt.Sprintf("secret/%s/%s", util.PodNamespace, GetSpecSecretName(appSlug))
-}
-
 func GetBundleCommand(appSlug string) []string {
 	redactURIs := []string{redact.GetKotsadmRedactSpecURI(), redact.GetAppRedactSpecURI(appSlug)}
 	redactors := strings.Join(redactURIs, ",")
 
 	command := []string{
 		"curl https://krew.sh/support-bundle | bash",
-		fmt.Sprintf("kubectl support-bundle %s --redactors=%s\n", GetSpecURI(appSlug), redactors),
+		fmt.Sprintf("kubectl support-bundle --load-cluster-specs --redactors=%s\n", redactors),
```

Comment:
Thinking about this more.  We are already [discovering specs](https://github.com/replicatedhq/kots/pull/3674/files#diff-3681d5e2735b3de144e2433bb7104c9bae6e47ddf4891d7d3017e9b7e2d83e5aR294) and [merging them](https://github.com/replicatedhq/kots/pull/3674/files#diff-3681d5e2735b3de144e2433bb7104c9bae6e47ddf4891d7d3017e9b7e2d83e5aR125) into `kotsadm-<app-slug>-supportbundle`.  Can we leave this as-is so that it would work with users running old versions of the support-bundle binary too?  It seems like `--load-cluster-specs` would not result in any additional collectors/analyzers.  What do you think?